Marlin/src/feature/fwretract.h

```diff
@@ -30,12 +30,11 @@
 #include "../inc/MarlinConfig.h"
 
 class FWRetract {
-private:
+  
+public:
   #if EXTRUDERS > 1
     static bool retracted_swap[EXTRUDERS];         // Which extruders are swap-retracted
   #endif
-
-public:
```

Comment:
because of the needed of compatibilty with fw when resuming ,   the statuses of FW must be public , to be copied , for the next tool after runout 
very important